_pages/views.md

```diff
@@ -172,6 +172,269 @@ vbox.
+"Example containers with 1 hbox and 2 vboxes, and a hbox underneath")
+
+If we only want ‘Date/time field 2’ to be seen when ‘Second time field’ 
+is populated, we can use visibility on ‘Date/time field 2’ like so:
+
+```xml
+
+<item labelAlign="left">
+	<textField binding="dateTime2" visible="hasSecondTime"/>
+</item>
+
+```
+
+With the visibility condition ‘hasSecondTime’ coming from a condition 
+in the ‘DocumentName.xml’ file, this condition will be inside the 
+Conditions section at the end of your Attributes section:
+
+```xml
+
+<conditions>
+    	<condition name="hasSecondTime">
+    	<description>True when this field has a second time field</description>
+    	<expression><![CDATA[(time2 != null)]]></expression>
+    	</condition>
+</conditions>
+
+```
+
+Now, without 'Time field 2' being populated, 'Date/Time field 2' is 
+not visible:
+
+![Visibility Example 1](../assets/images/views/Visibility.PNG 
+"Visibility example with field not showing")
+
+But once 'Time field 2' is populated and saved, 'Date/Time field 2' is visible:
+
+![Visibility Example 2](../assets/images/views/Visibility2.PNG 
+"Visibility example with field showing")
+
+#### Containers Example #3 - Tab Pane
+
+If we now wanted to further separate our top half (Left vbox and Right 
+vbox) and our second hbox underneath, we could use a tabpane, and place 
+the desired sections in separate tab. The edit view will now look like so:
+
+```xml
+
+<?xml version="1.0" encoding="UTF-8"?>
+<view xmlns="http://www.skyve.org/xml/view" 
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+		name="edit" title="DocumentName" 
+		xsi:schemaLocation="http://www.skyve.org/xml/view ../../../../schemas/view.xsd">
+    <tabPane>
+    	<tab title="First Tab">
+    		<hbox>
+		    	<vbox>
+		    		<form>
+					<column/>
+					<row>
+						<item>
+							<default binding="text1"/>
+						</item>
+					</row>
+					<row>
+						<item>
+							<default binding="date1"/>
+						</item>
+					</row>
+					<row>
+						<item>
+							<default binding="boolean1"/>
+						</item>
+					</row>
+					<row>
+						<item>
+							<default binding="time1"/>
+						</item>
+					</row>
+				</form>
+		    	</vbox>
+		    	<vbox>
+			    	<form border="true" borderTitle="Right vbox">
+			    		<column/>
+			    		<row>
+			    			<item labelAlign="left">
+			    				<default binding="text2"/>
+			    			</item>
+			    		</row>
+			    		<row>
+			    			<item labelAlign="left" required="true">
+			    				<default binding="date2"/>
+			    			</item>
+					</row>
+					<row>
+						<item labelAlign="left" label="SECOND TIME FIELD">
+							<default binding="time2"/>
+						</item>
+					</row>
+					<row>
+						<item labelAlign="left">
+							<default binding="dateTime1"/>
+						</item>
+					</row>
+					<row>
+						<item labelAlign="left">
+							<textField binding="dateTime2" visible="hasSecondTime"/>
+						</item>
+					</row>
+				</form>
+		    	</vbox>
+		</hbox>		
+    	</tab>
```

Comment:
Something weird is going on with the indentation in this example, can you try re-format it?